backend/FwLite/FwDataMiniLcmBridge/Api/FwDataMiniLcmApi.cs

```diff
@@ -742,20 +742,45 @@
         return await GetEntry(entry.Id) ?? throw new InvalidOperationException("Entry was not created");
     }
 
-    public Task<ComplexFormComponent> CreateComplexFormComponent(ComplexFormComponent complexFormComponent)
+    public Task<ComplexFormComponent> CreateComplexFormComponent(ComplexFormComponent complexFormComponent, BetweenPosition<ComplexFormComponent>? position = null)
     {
         UndoableUnitOfWorkHelper.DoUsingNewOrCurrentUOW("Create Complex Form Component",
             "Remove Complex Form Component",
             Cache.ServiceLocator.ActionHandler,
             () =>
             {
                 var lexEntry = EntriesRepository.GetObject(complexFormComponent.ComplexFormEntryId);
-                AddComplexFormComponent(lexEntry, complexFormComponent);
+                AddComplexFormComponent(lexEntry, complexFormComponent, position);
             });
         return Task.FromResult(ToComplexFormComponents(EntriesRepository.GetObject(complexFormComponent.ComplexFormEntryId))
             .Single(c => c.ComponentEntryId == complexFormComponent.ComponentEntryId && c.ComponentSenseId == complexFormComponent.ComponentSenseId));
     }
 
+    public Task MoveComplexFormComponent(Guid complexFormEntryId, ComplexFormComponent complexFormComponent, BetweenPosition<ComplexFormComponent> between)
```

Comment:
we can probably drop the `complexFormEntryId` parameter since it'll be in the component object